### DISCUSSION TITLE
Home Assistant integration

### DISCUSSION THREAD

**Discussion by T1mey** [SBFspot/SBFspot#605] [General]
Hello @all,

I have a SMA SB 5000TL-21 inverter with a webconnect module installed. It's as well connected to sunnyportal and delivers data to that service.
Wanna have the invertrer connected to homeassistant.
As there is a standard integration available I tried that... but always get connection issue during setup.

Shouldn't this work out of the box or are connections forbidden & blocked by SMA ?
Would SMAspot the tool to get my inverter connected ?

Is there a docker image available ?

**Comment by silliewimons** [comment]
by sillie,  there is a "less known" limit about SMAproducts, 
the "external-interface" from/to an SMAproduct supports only TWO external parallel channels - a third party has no acess,

from historically view that was no problem,  as long as interactionchannel-occupation remained short,  or  few-candidate users
that has changed:
 - since SMAproducts needed more/permanent inter-actions in view of extended functions,
 - for that reason the limit for TWO-parallel accesses is now a bottle-neck   when other #party want to join system-communication 
 - remember -priority-needed for channels-  occupied by "energy-meter&data-manager" / webconnect
   leaving smaller-corridor for: -sunny-portal / -home-assistant / -others . . .  

kr

**Comment by ziogref** [comment]
I use "SMA Solar" in Home assistant that takes info from my SMA Sunnyboy 6 and put into into the solar dashboard. That includes my usage data from my Home Manager 2.0.

https://www.home-assistant.io/integrations/sma

Note: This has nothing to do with this project, it's just what I use to get my info into HA.
